### ISSUE TITLE
Accessors.plot() fails with invalid reference to common_features from Carpenter Workshop.

### ISSUE THREAD

**Issue by rodinia814** [blaylockbk/Herbie#186]
I have both the latest Herbie and the latest Carpenter Workshop installed.

The plot() method in accessors.py references common_features of toolbox.cartopy_tools in Carpenter Workshop.  But that was renamed to EasyMap recently.  As a result, getting invalid reference errors at runtime.

https://github.com/blaylockbk/Herbie/blob/main/herbie/accessors.py#L309

  File "/usr/local/lib/python3.9/site-packages/herbie/accessors.py", line 358, in plot
    ax = common_features(**common_features_kw).STATES().ax
UnboundLocalError: local variable 'common_features' referenced before assignment

**Comment by AmeerArsala** [comment]
same here

**Comment by blaylockbk** [comment]
Sorry about that. I merged #270 and hope that fixes the problem. Note: `ds.herbie.plot()` is not implemented very well, and could use some work.

Here are a few examples:

```python
from herbie import Herbie
ds = Herbie("2024-01-01").xarray("(?:TMP|DPT):2 m")
ds.herbie.plot()
```
![image](https://github.com/blaylockbk/Herbie/assets/6249613/5eaa9a4b-f780-461f-982b-b54a66fd79e9)

![image](https://github.com/blaylockbk/Herbie/assets/6249613/e320764f-676d-43cb-abd9-0a93e5d88b21)

```python
ds = Herbie("2024-01-01").xarray("HGT:500 mb")
ds.herbie.plot()
```
![image](https://github.com/blaylockbk/Herbie/assets/6249613/13636d39-165c-4527-b0e2-2693dbab888a)



```python
ds = Herbie("2024-01-01").xarray("REFC:entire")
ds.herbie.plot()
```
![image](https://github.com/blaylockbk/Herbie/assets/6249613/985e5915-9ec8-481d-8882-b320caa5ce0d)

```python
ds = Herbie("2024-01-01").xarray("(?:HGT|LAND):surface")
ds.herbie.plot()
```
![image](https://github.com/blaylockbk/Herbie/assets/6249613/1d595544-aaa0-4142-a98d-526153a42734)

![image](https://github.com/blaylockbk/Herbie/assets/6249613/9e7afa67-abb4-49a5-a400-de664a381d39)


```python
ds = Herbie("2024-01-01", model="gfs").xarray("HGT:500 mb")
ds.herbie.plot()
```
![image](https://github.com/blaylockbk/Herbie/assets/6249613/7c87f50e-0729-41cf-ba67-20e09d104215)
